store/orm/orm.go

```diff
@@ -707,14 +707,14 @@ func (orm *ORM) CreateInitiator(initr *models.Initiator) error {
 }
 
 // SaveHead saves the indexable block number related to head tracker.
-func (orm *ORM) SaveHead(n *models.IndexableBlockNumber) error {
+func (orm *ORM) SaveHead(n *models.Head) error {
 	return orm.DB.Save(n).Error
 }
 
-// LastHead returns the last ordered IndexableBlockNumber.
-func (orm *ORM) LastHead() (*models.IndexableBlockNumber, error) {
-	number := &models.IndexableBlockNumber{}
-	err := orm.DB.Order("digits desc, number desc").First(number).Error
+// LastHead returns the last Head in descending order.
```

Comment:
`returns the last Head in descending order` this doesn't make any sense, a single head has no order.

Reply:
Open to a suggestion.

How about
```
LastHead returns the most recent head entry.
```
?